packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -370,6 +370,13 @@ function checkSortByKey(sortBy: any): void {
   }
 }
 
+function sortByOptionToString(sortBy: any) {
+  if (Array.isArray(sortBy)) {
+    return `${sortBy[0]} ${sortBy[1] ? 'ASC' : 'DESC'}`;
+  }
+  return `${sortBy} DESC`;
+}
```

Comment:
That function won't work well with the current iOS implementation. It won't be understood by https://github.com/expo/expo/blob/9df607dd0286e61f7606253b2941924db4cbae9a/packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m#L1111-L1130